apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -349,97 +362,119 @@ interface SectionTableBodyProps {
     scheduleNames: string[];
 }
 
-//TODO: SectionNum name parity -> SectionNumber
+const tableBodyCells: Record<SectionTableColumn, React.ComponentType<any>> = {
+    sectionCode: CourseCodeCell,
+    sectionDetails: SectionDetailsCell,
+    instructors: InstructorsCell,
+    dayAndTime: DayAndTimeCell,
+    location: LocationsCell,
+    sectionEnrollment: SectionEnrollmentCell,
+    restrictions: RestrictionsCell,
+    status: StatusCell,
+};
+
+// TODO: SectionNum name parity -> SectionNumber
 const SectionTableBody = withStyles(styles)((props: SectionTableBodyProps) => {
     const { classes, section, courseDetails, term, colorAndDelete, highlightAdded, scheduleNames } = props;
+
     const [addedCourse, setAddedCourse] = useState(colorAndDelete);
 
-    const [scheduleConflict, setScheduleConflict] = useState(false);
+    const [calendarEvents, setCalendarEvents] = useState(AppStore.getCourseEventsInCalendar());
+
+    const [activeColumns, setColumns] = useState(RightPaneStore.getActiveColumns());
+
+    /**
+     * Additional information about the current section being rendered.
+     */
+    const sectionDetails = useMemo(() => {
+        return {
+            // If there already exists a more well-written way to translate secton.meetings days (string) into a number, LMK
+            // Converts SuTuTh -> [Su, Tu, Th] -> [0, 2, 4]
+            day: section.meetings[0].days.match(/[A-Z][a-z]*/g)?.map((day: string) => DAYS_TO_NUMS[day]),
+            translatedTime: translateWebSOCTimeTo24HourTime(section.meetings[0].time),
+            startTime: '',
+            endTime: '',
+        };
+    }, [section.meetings]);
```

Comment:
Instead of recalculating this arbitrarily, we know that it should be updated whenever `section.meetings` changes, so memoize it with that